docs/website/content/docs/v0.3/Configuration/serverclasses.md

```diff
@@ -31,3 +31,5 @@ spec:
 ```
 
 Servers would only be added to the above class if they had _EITHER_ CPU info, _AND_ the label associated with the server resource.
+
+Additionally, Sidero automatically creates and maintains a server class called `"all"` that includes all (accepted) servers.
```

Comment:
Should it be "all" or "any"?

Reply:
Any sounds like a better name to me, yes.